### PR TITLE
Modified TestMerge.cpp

### PR DESCRIPTION
**Pull request by Lemmings19**
This is the description of the changes I made.

### CODE REVIEW

**Comment by Lemmings19** [comment]
Can you please review my code and let me know if you think anything should change @Varkoff

File: testMerge/testMerge.cpp [hunk 1]
```diff
@@ -9,4 +9,5 @@ int main()
 	int a = 2;
 	int b = a + a;
 	cout << b << endl;
+	cout << "Hello world" << endl;
```
Comment (Lemmings19) [inline testMerge/testMerge.cpp]: What do you think of this @Varkoff? Is it good enough?

Comment (Lemmings19) [inline testMerge/testMerge.cpp]: What do you think of this @Varkoff? Is it good enough?

Comment (Varkoff) [reply testMerge/testMerge.cpp]: Looks amazing ! Thanks partner :) @Lemmings19

Comment (Varkoff) [reply testMerge/testMerge.cpp]: Looks amazing ! Thanks partner :) @Lemmings19

**Pull request merged**